pom.xml

```diff
@@ -1784,5 +1784,38 @@
                 </plugins>
             </build>
         </profile>
+        <profile>
+            <id>ci</id>
+            <properties>
+                <!-- the *local* master, not refs/remotes/... -->
+                <gib.referenceBranch>master</gib.referenceBranch>
+            </properties>
+            <build>
+                <plugins>
+                    <plugin>
+                        <groupId>com.vackosar.gitflowincrementalbuilder</groupId>
+                        <artifactId>gitflow-incremental-builder</artifactId>
+                        <version>3.14.4</version>
+                        <extensions>true</extensions>
+                        <configuration>
+                            <disableIfBranchMatches>master</disableIfBranchMatches>
+                            <compareToMergeBase>true</compareToMergeBase>
+                            <uncommitted>true</uncommitted>
+                            <untracked>true</untracked>
+                            <excludePathsMatching>(\.github([/\\].*)?|\.mwn([/\\].*)?|mvnw|README.md|LICENSE)</excludePathsMatching>
```

Comment:
What's `.mwn`?

Reply:
That's an awful typo, it was supposed to be the `.mvn` dir.